tensorflow_model_optimization/python/core/quantization/keras/quantize_aware_activation.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, activation, quantizer, step, quantize_wrapper):
     self._min_post_activation, self._max_post_activation = \
       self._add_range_weights('post_activation')
 
+  @property
+  def __name__(self):
+      return self.activation.__name__
```

Comment:
self.activation won't always have the __name__ property. This function should check for that.

See _is_keras_builtin_activation.